### PR TITLE
Fix help() rpc to display new lines properly be default in snippet sh…

### PR DESCRIPTION
**Pull request by winterfroststrom**
…ell.

---
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/546)

### CODE REVIEW

File: tools/snippet_shell.py [hunk 1]
```diff
@@ -34,13 +34,30 @@
 from mobly.controllers.android_device_lib import jsonrpc_shell_base
 
 
+def _patch_print_rpc(snippet, name):
```
Comment (xpconanfan) [inline tools/snippet_shell.py]: i was thinking simply defining the `help` method in the snippet client class...
would be less hacky?

Comment (winterfroststrom) [reply tools/snippet_shell.py]: Doesn't that potentially cause issues if someone actually wants the help() response?
I could add a default argument to disable printing in that case I guess.

Comment (winterfroststrom) [reply tools/snippet_shell.py]: Changed.

This is a bit more dangerous to do, but it's probably okay.

File: mobly/controllers/android_device_lib/snippet_client.py [hunk 2]
```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 """JSON RPC interface to Mobly Snippet Lib."""
 
+from __future__ import print_function
```
Comment (xpconanfan) [inline mobly/controllers/android_device_lib/snippet_client.py]: `print_function` is not used?
is this a magical line?

Comment (winterfroststrom) [reply mobly/controllers/android_device_lib/snippet_client.py]: it's for python2 compatibility?
print vs print()

**Pull request merged**